src/KubernetesClient.Informers/Extensions.cs

```diff
@@ -0,0 +1,174 @@
+using System;
+using System.Collections.Generic;
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using k8s.Models;
+using Microsoft.Rest;
+
+namespace k8s.Informers
+{
+    public static class Extensions
```

Comment:
I'd make this internal.

Reply:
There are a few methods in there that are meant to be public, mainly `DisposeWith` and `CombineWith` to help in building up RX code. `Watch` is also useful for anyone that wants to consume raw Watcher output as observable. I've already switch `polyfill` type code (ToHashset, etc) to be internal.